### PR TITLE
OSDOCS-11208-dev-console: Documented dev console bug fixes 4.17

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.17

Issue:
[OSDOCS-11208](https://issues.redhat.com/browse/OSDOCS-11208)

Link to docs preview:
* [Developer Console](https://82532--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-17-release-notes.html#ocp-4-17-bug-fixes_release-notes)

### CODE REVIEW

File: release_notes/ocp-4-17-release-notes.adoc [hunk 1]
```diff
@@ -1042,6 +1042,18 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
...
+
+* Previously, s2i build strategy was not explicitly mentioned in the `func.yml`. Therefore you could not create {ServerlessProductName} functions with the repository. Additionally, error messages were not available if s2i is not mentioned or if `func.yml`. As a result, identifying the reason of failures was not apparent. With this update, if the s2i build strategy is not mentioned, users can still create a function. If it is not s2i, users cannot create a function. The error messages are now different for both the cases. (link:https://issues.redhat.com/browse/OCPBUGS-33733[*OCPBUGS-33733*])
+
+* Previously, when using a Quick Start guided tour in the {product-title} web console, it took multiple clicks of the *Next* button in order to skip to the next step if the `check your work` dialog was ignored. With this update, it only takes one click, regardless of the state of the `check your work` box. (link:https://issues.redhat.com/browse/OCPBUGS-25929[*OCPBUGS-25929*])
```
Comment (tmalove) [inline release_notes/ocp-4-17-release-notes.adoc:1049]:
```suggestion
* Previously, when using a Quick Start guided tour in the {product-title} web console, it took multiple clicks of the *Next* button to skip to the next step if the `check your work` dialog was ignored. With this update, it only takes one click, regardless of the state of the `check your work` box. (link:https://issues.redhat.com/browse/OCPBUGS-25929[*OCPBUGS-25929*])
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

File: release_notes/ocp-4-17-release-notes.adoc [hunk 2]
```diff
@@ -1042,6 +1042,18 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
...
+
+[discrete]
+[id="ocp-4-17-driver-toolkit-bug-fixes_{context}"]
+==== Driver Tool Kit (DTK)
```
Comment (tmalove) [inline release_notes/ocp-4-17-release-notes.adoc:1053]:
```suggestion
==== Driver Toolkit (DTK)
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#t

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-17-release-notes.adoc (hunk 1 above)
Comment (dfitzmau) [reply release_notes/ocp-4-17-release-notes.adoc:1049]: :heavy_check_mark:

File: release_notes/ocp-4-17-release-notes.adoc (hunk 2 above)
Comment (dfitzmau) [reply release_notes/ocp-4-17-release-notes.adoc]: :heavy_check_mark:

**Pull request merged**